### ISSUE TITLE
Druid - Clearcasting + Ferocious Bite

### ISSUE THREAD

**Issue by fnm86** [SunwellTracker/issues#3434]
Decription: 
The Clearcasting state reduces the Mana, Rage or Energy cost of your next damage, healing spell or offensive ability by 100%.
But did not work on Ferocious Bite - when we cast it in Clearcasting anyway takes energy.

How it works: 
Ferocious Bite used on Clearcasting active takes energy,

How it should work: 
Ferocious Bite used on Clearcasting should not take energy,

Source (you should point out proofs of your report, please give us some source): 

Spell description:
"Each of the Druid's damage, healing spells and auto attacks has a chance of causing the caster to enter a Clearcasting state. The Clearcasting state reduces the Mana, Rage or Energy cost of your next damage, healing spell or offensive ability by 100%."

**Comment by Soufly** [comment]
https://wow.gamepedia.com/Ferocious_Bite

**Comment by styxel** [comment]
Yep ooc only makes the base fb cost 35 - > 0 energy but it stil uses 1-30 for extra damage making it still cost energy on ooc, fb is working correctly on the server.
